weDevsOfficial/dokan
# 🐛 Fix: dummy data serialize security issue

### All Submissions:






### Changes proposed in this Pull Request:

Dummy data serialize data security issue.

Closes #

### How to test the changes in this Pull Request:

1. Test dummy date create
2. check if all data saved successfully
3. check account passeord works, for all dummy vendor account password is `123456789`



### Changelog entry

> Dummy data serialize data security issue.


### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.


### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.
